hypervisor/arch/x86/Kconfig

```diff
@@ -41,14 +41,14 @@ config PARTITION_MODE
 endchoice
 
 config PLATFORM
-	string
+	string "The type of boot firmware (BIOS) on the target platform"
```

Comment:
Small warning here:

Putting a prompt on a symbol makes it configurable in the `menuconfig` interface, so this will make it possible to select e.g. `PLATFORM_UEFI` and then change `PLATFORM` to something other than "uefi".

Maybe it's intentional, but just in case it was meant as a help text. :)

I added [a page with some Kconfig tips](https://docs.zephyrproject.org/latest/application/kconfig-tips.html) to the Zephyr documentation btw. The prompt vs. no-prompt thing pretty subtle.

Reply:
Thanks for pointing this out @ulfalizer ! It was not my intention to make this user-configurable. I merely wanted some text to appear when we generate the HTML documentation in the symbol description table ([here](https://projectacrn.github.io/latest/reference/kconfig/index.html)).

I did not find an easy way to do this without making it user-configurable so I'll just remove the prompt.

Looking at the `PLATFORM` config option, I realize it's pretty obsolete nowadays, we know everything we need using `CONFIG_PLATFORM_UEFI` or `CONFIG_PLATFORM_SBL`... so I'm proposing an extra change in the PR to remove it completely (the hypervisor `Makefile` was changed to accomodate this removal).

Reply:
Yeah, maybe `genrest.py` could be tweaked to fall back on the (beginning of the) help text on the index page if there's no prompt. A third column might work as well.

No problem!